frontend/src/views/pages/dealerships/Dealerships.vue

```diff
@@ -0,0 +1,176 @@
+              Start your experience by creating your very first dealership by
+              click the button below.</CAlert
+            >
+            <CAlert
+              v-if="
+                $store.state.auth.createDealershipCompleted &&
+                !$store.state.auth.createUserCompleted
+              "
+              color="primary"
+              >Good job on creating your very first dealership! Now to get your
+              team started, select your dealership and start building your
+              staff.</CAlert
+            >
+          </div>
+        </CRow>
+        <CRow class="mb-3">
+          <div class="col-sm-auto">
+            <CButton
+              color="primary"
+              id="add-new-dealership"
+              @click="
+                () => {
+                  addingDealership = true;
+                }
+              "
+              >Add New Dealership</CButton
+            >
+          </div>
+          <div class="col-sm-auto"></div>
+        </CRow>
+        <CCard>
+          <CCardHeader>
+            <slot name="header"> List of registered dealerships </slot>
+          </CCardHeader>
+          <CCardBody>
+            <CDataTable
+              :fields="tableFields"
+              :items="tableItems"
+              :striped="true"
+              :items-per-page="10"
+              :fixed="true"
+              :clickable-rows="true"
+              @row-clicked="clickRow"
+            >
+              <template #admin="{ item }">
+                <td>{{ item.admin.first_name }} {{ item.admin.last_name }}</td>
+              </template>
+              <template #name="{ item }">
+                <td>
+                  {{ item.name }}
+                  <CBadge
+                    color="primary"
+                    v-if="
+                      $store.state.auth.createDealershipCompleted &&
+                      !$store.state.auth.createUserCompleted
+                    "
+                    :style="{
+                      'vertical-align': 'middle',
+                      'margin-top': '-0.5em',
+                    }"
+                    >NEW!</CBadge
+                  >
+                </td>
+              </template>
+            </CDataTable>
+          </CCardBody>
+        </CCard>
+      </CCol>
+    </CRow>
+    <CModal
+      :show.sync="addingDealership"
+      :centered="true"
+      title="Modal title 2"
+      size="lg"
+    >
+      <dealership-add />
+      <template #header>
+        <h6 class="modal-title">Create a new dealership!</h6>
+        <CButtonClose @click="addingDealership = false" />
+      </template>
+      <template #footer>
+        <!-- <CRow class="justify-content-center">
+        </CRow>
+        <CButton @click="addingDealership = false" color="danger">Discard</CButton>
+        <CButton @click="addingDealership = false" color="success">Accept</CButton> -->
+        <span></span>
```

Comment:
Remove commented out code

Reply:
Fixed in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/37/commits/8c83bea65e9c8a6186a069f6415340392a55a7b4)